API.md

```diff
@@ -95,6 +95,15 @@ const db = await orbitdb.keyvalue('profile')
 
 ## Public Instance Methods
 
+Before to start you've to know that OrbitDB has different types of databases. Each one made to satisfy a different purpose. The databases that you can create are:
```

Comment:
Before starting, you should know that OrbitDB has different types of databases. Each one satisfies a different purpose. The databases that you can create are:

Reply:
> 
> 
> Before starting, you should know that OrbitDB has different types of databases. Each one satisfies a different purpose. The databases that you can create are:

ok thanks for the feedback I'll commit the changes